no/auth.json

```diff
@@ -25,6 +25,16 @@
       "country_input_error_invalid": "Et gyldig land er påkrevd",
       "choose_another_method_link": "Velg en annen metode"
     },
+    "mfa_email_page": {
+      "page_title": "Skriv inn e-post | ${business_name}",
+      "heading": "Autentiser via e-post",
```

Comment:
"Autentiser" is not a common word for non-technical people in Norway (typically our end users, at least), so it would probably be better to say "Logg inn via e-post" or something like that for this translation. But since I'm not sure of the context this is shown in, and the mfa_sms_page translations also used "autentiser", I'll just use it to keep consistency :)